journal/modernized/16-chapter-13.adoc

```diff
@@ -415,6 +445,9 @@ it was concluded that a committee should be
 appointed to express the sense of the meeting,
 in an endorsement of my certificate.
 
+--
+//Not sure what to do here either. It's from the same letter, but idk if the offset is enought to seperate it? 
```

Comment:
yeah, kind of tough.  let's re-open another embedded document _below_ the offset.  it won't need any salutation or context-open, etc.  And then close it right before `Sheffield, Tenth Month 8th.`